service/src/main/java/uk/nhs/adaptors/gp2gp/mhs/InboundMessageConsumer.java

```diff
@@ -19,7 +19,7 @@
     private final InboundMessageHandler inboundMessageHandler;
     private final MDCService mdcService;
 
-    @JmsListener(destination = "${gp2gp.amqp.inboundQueueName}")
+    @JmsListener(destination = "${gp2gp.amqp.inboundQueueName}", concurrency = "10")
```

Comment:
Up for discussion - could we make that parameter configurable? This could allow us to tweak the app if we spot problems during perf tests quicker.